omod/src/main/java/org/openmrs/module/fhir/providers/RestfulPatientResourceProvider.java

```diff
@@ -182,7 +182,7 @@ public MethodOutcome createFHIRPatient(@ResourceParam Patient patient) {
 		MethodOutcome retVal = new MethodOutcome();
 		retVal.setId(new IdDt(FHIRConstants.PATIENT, patient.getId().getIdPart()));
 		OperationOutcome outcome = new OperationOutcome();
-		outcome.addIssue().setDetails("Patient is successfully created");
+		outcome.addIssue().setDetails("Patient is successfully created with " + patient.getId().getIdPart());
 		retVal.setOperationOutcome(outcome);
 		return retVal;
```

Comment:
@surangak  @harsha89 see the latest changes. related ticket https://issues.openmrs.org/browse/FM-95